INSTALL.md

```diff
@@ -8,7 +8,7 @@ How to install application using helm:  <br />
     helm install ReleaseName ChartName
     
     a.) Add helm repository in tractusx:
-           helm repo add sd-factory https://eclipse-tractusx.github.io/charts/dev
+           helm repo add sd-factory https://eclipse-tractusx.github.io/charts/tractusx-dev
```

Comment:
Is this working? Have you tested it? Naming the repo `sd-factory` and then on line [13](https://github.com/eclipse-tractusx/sd-factory/pull/38/files#diff-09b140a43ebfdd8dbec31ce72cafffd15164d2860fd390692a030bcb932b54a0R13) searching repo with name `tractusx-dev` seems wrong to me.

Reply:
This has been fixed